.idea/codeStyleSettings.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This deserves a separate commit, but I can do the commit-splitting later if you prefer.

This is for IntelliJ?